### ISSUE TITLE
Editing The Struggle of Maintaining Top YouTuber Status 

### ISSUE THREAD

**Issue by ZachAndrews98** [nadeld/nadeld.github.io#3]
First Paragraph:
The last sentence is lacking and ends on a sort of cliffhanger, finalize the thought that you had during that paragraph.
Second Paragraph:
Continue the track that it is on and develop it more.
Additional:
Your TL;DR is fairly long, I would reword it to make it more concise.
Your header image and title text are fairly similar in color which makes the title hard to read, maybe find a different header or change the color of the text.

**Comment by nadeld** [comment]
Thanks Zach! Currently working on making changes

**Comment by ZachAndrews98** [comment]
Paragraph one:
- change "great source for people viewing content" to "great source for people to view content".
- change "adblock" to AdBlock.
- "Many viewers use adblock because they do not want to waste 20 seconds for each video they watch and this 20 seconds is ironic because if they really enjoy watching their youtuber you would think they would sit through the ad to support them"- this sentence runs on and could be reworded a lot better
- change "stand out over the competition" to "stand out from the competition".
- FIX LAST SENTENCE, very open ended and doesn't state what is combined to create the grueling task.

Paragraph two:
- fix the first sentence (first four lines of the second paragraph), runs on and on.
- fix spelling of committment (commitment).
- put "" around the titles of the DSP series.
- fix the sentence that talks about income in February, is very wordy and at times confusing.

Paragraph three:
fix the second sentence, wording is very redundant "paid-subscription $9.99 per month service".
- fix spelling of deleivering (delivering).
- fix spelling of plauging (plaguing).

Paragraph four:
- fix "they could have many people watch"- badly worded
- change "YouTuber’s" to "YouTubers"
- fix "half of total channels" to "half of the total channels"
- don't know what this sentence is trying to say: "Based of this growing trend, the average viewer will continue to be provided with so much content and benefit them in ways that unfortunately uploaders who do YouTube as a living a complete opposite feeling"
- fix "never give up due the current"
- continue work on the rest of the article
